frontend/src/components/admin/assessment-creation/AssessmentEditorHeader.tsx

```diff
@@ -134,14 +134,14 @@ const AssessmentEditorHeader = ({
             >
               Publish
             </ActionButton>
-            <Popover>
-              <VStack divider={<Divider />} spacing="0em">
-                {isEditing && (
+            {isEditing && (
+              <Popover>
+                <VStack divider={<Divider />} spacing="0em">
                   <PopoverButton name="Archive" onClick={onArchiveModalOpen} />
-                )}
-                <PopoverButton name="Delete" onClick={onDeleteModalOpen} />
-              </VStack>
-            </Popover>
+                  <PopoverButton name="Delete" onClick={onDeleteModalOpen} />
```

Comment:
i think design originally wanted delete option, in which if its a new assessment the editor just closes. i'm fine w getting rid of it - but can we clean up [this line](https://github.com/uwblueprint/jump-math/blob/staging/frontend/src/components/admin/assessment-creation/AssessmentEditorHeader.tsx#L159) then? and remove `handleCloseEditor`